### DISCUSSION TITLE
使用 PaoPaoDNS+sing-box 求助

### DISCUSSION THREAD

**Discussion by MiWRT** [kkkgo/PaoPaoDNS#184] [问答和帮助]
先弄好 PaoPaoDNS后，想搭配使用 sing-box ，弄好 sing-box 后电脑网关指向 sing-box 的地址，出墙什么的都正常。
然后配置 PaoPaoDNS 配置文件里 CUSTOM_FORWARD= sing-box IP 地址：53，出墙就不正常了，请问 sing-box 这台透明网关，还需要设置什么？感谢

**Comment by kkkgo** [comment]
建议参考其他人的教程和拓扑，https://github.com/kkkgo/PaoPaoGateWay/discussions/categories/%E9%85%8D%E7%BD%AE%E5%88%86%E4%BA%AB  
你想要用CUSTOM_FORWARD分流的话拓扑就是静态路由+FAKEDNS，不是随便指个网关就可以用的，你网关DNS必须给出的是fakeip。
